### PR TITLE
chore: remove non-functional question mark from text

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/9002019994/PRF-6310
- remove non-functional question mark from text
- link feedback form with help-button

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6310 On cysync hide question marks and add support page in "Help ?" button](https://app.clickup.com/t/9002019994/PRF-6310)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6310 On cysync hide question marks and add support page in "Help ?" button](https://app.clickup.com/t/9002019994/PRF-6310)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6310 On cysync hide question marks and add support page in "Help ?" button](https://app.clickup.com/t/9002019994/PRF-6310)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6310 On cysync hide question marks and add support page in "Help ?" button](https://app.clickup.com/t/9002019994/PRF-6310)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6310 On cysync hide question marks and add support page in "Help ?" button](https://app.clickup.com/t/9002019994/PRF-6310)

File: packages/cysync-core/src/constants/i18n/en.ts [hunk 1]
```diff
@@ -1100,7 +1100,7 @@ const en = {
                 {
                   // TODO: add functionality to 
```
Comment (irshadCypherock) [inline packages/cysync-core/src/constants/i18n/en.ts]: Remove TODO's as well

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/constants/i18n/en.ts (hunk 1 above)
Comment (ujjwal-cyph) [reply packages/cysync-core/src/constants/i18n/en.ts]: removed

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**